docs/en/operation/StressBench.md

```diff
@@ -263,3 +263,154 @@ Here is a sample usage demo, where we have one master and three workers:
+
+<table class="table table-striped">
+    <tr>
+        <th>Parameter</th>
+        <th>Default Value</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td>concurrency</td>
+        <td>2</td>
+        <td>The number of simulated clients on each node. Typically benchmarks use 1 thread for 1 
+        simulated client.</td>
+    </tr>
+    <tr>
+        <td>duration</td>
+        <td>5s</td>
+        <td>How long the benchmark should be recording measurements for.</td>
+    </tr>
+</table>
+
+### Available benchmarks
+
+These RPC benchmarks are currently available:
+
+- [`GetPinnedFileIdsBench`](#getpinnedfileidsbench)
+- [`RegisterWorkerBench`](#registerworkerbench)
+- [`WorkerHeartbeatBench`](#workerheartbeatbench)
+- [`StreamRegisterWorkerBench`](#streamregisterworkerbench)
+
+#### `GetPinnedFileIdsBench`
+
+For the `GetPinnedFileIds` RPC.
+
+This benchmark calls the RPC repeatedly until duration is reached.
+
+`GetPinnedFileIds` is periodically called by block workers on every heartbeat to poll the 
+pinned file list. It can be expensive if the pinned file list is huge. 
+
+Parameters:
+
+<table class="table table-striped">
+    <tr>
+        <th>Parameter</th>
+        <th>Default Value</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td>num-files</td>
+        <td>100</td>
+        <td>Number of pinned files. Use this parameter to adjust the load incurred by the RPC. 
+            Higher numbers incur higher loads.</td>
+    </tr>
+    <tr>
+        <td>base-dir</td>
+        <td>/get-pin-list-bench-base</td>
+        <td>The temporary directory to contain the test files.</td>
+    </tr>
+</table>
+
+#### `RegisterWorkerBench` 
+
+For the `RegisterWorker` RPC and the `RequestRegisterLease` RPC.
+
+This benchmark calls the RPC only once.
+
+This benchmark measures the performance of the process of new worker registration. 
+The`RegisterWorker` RPC carries a list of blocks that is currently stored in the worker's storage. 
+This RPC can be expensive if the list is huge. 
+The `RequestRegisterLease` RPC is part of the mechanism that the master uses to 
+control the number of concurrently registering clients so that it does not exceed the master's 
+processing capacity.
+
+Parameters:
+
+<table class="table table-striped">
+    <tr>
+        <th>Parameter</th>
+        <th>Default Value</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td>tiers</td>
+        <td>n/a</td>
+        <td>A semicolon-separated list of storage tiers. Each tier is a comma-separated 
+            list of numbers of blocks in directories on that tier. At most 3 tiers are supported. 
+            Example: `100,200,300;1000,1500;2000`</td>
+    </tr>
+</table>
+
+#### `WorkerHeartbeatBench`
+
+For the `WorkerHeartbeat` RPC.
+
+`WorkerHeartbeat` is called on every worker heartbeat. Similar to the `RegisterWorker` RPC, this 
+RPC carries a block list, and can be expensive if that list is huge.
```

Comment:
```suggestion
`WorkerHeartbeat` is called on every worker heartbeat. This RPC carries the block update on this worker since the last heartbeat. If the workers at your scale are under high load, the heartbeat message will also be large and incurs significant overhead on the master side.
```